Fix notation, formulas and cites in scalars in Background section (#178)
chhoumann/thesis-chemcam: report_thesis/src/sections/background/preprocessing/index.tex

In this subsection, we discuss the preprocessing methods used in our machine learning pipeline.

> We cover various normalization techniques such as Z-score normalization, max absolute scaling, min-max normalization, robust scaling, Norm 3, power transformation, and quantile transformation.

Reviewer: ```suggestion
We cover the following normalization techniques: Z-score normalization, max absolute scaling, min-max normalization, robust scaling, Norm 3, power transformation, and quantile transformation.
```
Feels odd to such "such as" for an exhaustive list.

Author: lol
